app/helper/helper.go

```diff
@@ -31,3 +33,21 @@ func ToBigInt(value string) (*big.Int, error) {
 
 	return amount, nil
 }
+
+// ToMirrorNodeTransactionID parses TX with format `0.0.X@{seconds}.{nanos}` to format `0.0.X-{seconds}-{nanos}`
+func ToMirrorNodeTransactionID(txId string) string {
+	split := strings.Split(txId, "?")
+	split = strings.Split(split[0], "@")
+	accId := split[0]
+	split = strings.Split(split[1], ".")
+	return fmt.Sprintf("%s-%s-%s", accId, split[0], fmt.Sprintf("%09s", split[1]))
```

Comment:
`split[0]` -> `fmt.Sprintf("%09s", split[0])`
The same issue happens with the seconds sometimes as well.

Reply:
If you knew what is the amount behind split[0], you would know it is not possible to be less than 10 symbols